assets/js/nebula.js

```diff
@@ -2728,23 +2728,34 @@ nebula.cf7LocalStorage = function(){
 		return false;
 	}
 
-	jQuery('.wpcf7-textarea, .wpcf7-text').each(function(){
+	jQuery('.wpcf7-textarea, .wpcf7-text, .wpcf7-select, .wpcf7-checkbox input').each(function(){
 		var thisLocalStorageVal = localStorage.getItem('cf7_' + jQuery(this).attr('name'));
 
-		//Fill textareas with localstorage data on load
+		//Complete inputs with localstorage data on load
 		if ( !jQuery(this).hasClass('do-not-store') && !jQuery(this).hasClass('.wpcf7-captchar') && thisLocalStorageVal && thisLocalStorageVal !== 'undefined' && thisLocalStorageVal !== '' ){
-			if ( jQuery(this).val() === '' ){ //Don't overwrite a field that already has text in it!
+			if( jQuery(this).attr('type') == 'checkbox' ){
+				// local storange vals are stored as strings but the .prop method requires a bool
+				if( thisLocalStorageVal == 'true' ){
+					jQuery(this).prop('checked', true);
```

Comment:
What if instead of this conditional if/else we did something like:

```js
thisLocalStorageVal = (thisLocalStorageVal.toLowerCase() === 'true'); //Convert string to boolean
jQuery(this).prop('checked', thisLocalStorageVal);
```

Reply:
I want to note that I was two 's away from doing this... I `===`'d against `true` instead of `'true'`and when I didn't work I said welp on to the next method.

Reply:
Ah that probably would have worked with `==` BUT `"false" == true` so might have been some weird edge cases.